### DISCUSSION TITLE
Alarm clock or delay play

### DISCUSSION THREAD

**Discussion by SCMusson** [PaulWoitaschek/Voice#2132] [Ideas]
I have an alarm clock that wakes me up to the radio. I'd love to wake up to the audio book in the morning. Either the ability to set a time when the audio book starts playing or having the option to delay playback so I could set it to say start playing 8 hours from now.

Scrolled for quite a while couldn't see anything that jumped out as being similar to this

**Comment by makiapplpretzele** [comment]
I think an Intent for playing a specific book + an alarm app can most likely do this
